gst/nnstreamer/tensor_query/tensor_query_serversrc.c

```diff
@@ -173,12 +172,12 @@ gst_tensor_query_serversrc_finalize (GObject * object)
   src->operation = NULL;
   g_free (src->broker_host);
   src->broker_host = NULL;
-  gst_tensors_config_free (&src->src_config);
   if (src->server_data) {
     nnstreamer_query_server_data_free (src->server_data);
     src->server_data = NULL;
   }
   G_OBJECT_CLASS (parent_class)->finalize (object);
+  gst_caps_replace (&src->caps, NULL);
```

Comment:
move this to line 179 (or, is it possible removing src->caps?)

Reply:
Removed. thanks.